examples/tutorials/hello-gltf/app.ts

```diff
@@ -109,17 +111,18 @@ const lightSources: LightingProps = {
     type: 'ambient'
   },
   directionalLights: [
+    // @ts-expect-error Remove once npm package updated with new types
     {
       color: [222, 244, 255],
       direction: [1, -0.5, 0.5],
       intensity: 10,
-      position: [0, 0, 0],
       type: 'directional'
     }
   ],
   pointLights: [
     {
-      attenuation: 0,
+      // @ts-expect-error Remove once npm package updated with new types
+      attenuation: [1, 0, 0],
```

Comment:
Is the intention to explicitly override defaults here? Or are the defaults not working?

Reply:
When the full build runs, this line triggers an error because the code is picking up the types from the currently published package

Reply:
I was just checking if it works, have removed the line